contrib/endpoints/src/api_manager/check_auth.cc

```diff
@@ -213,7 +213,9 @@ void AuthChecker::LookupJwtCache() {
   if (cache_hit) {
     CheckAudience(true);
   } else {
+    env_->LogInfo("Before Parse JWT");
     ParseJwt();
+    env_->LogInfo("After Parse JWT");
```

Comment:
Looks like the merge put them back in. I will delete them before the next commit.